src/QsCompiler/CompilationManager/EditorSupport/EditorCommands.cs

```diff
@@ -40,10 +40,10 @@ public static SymbolInformation[] DocumentSymbols(this FileContentManager file)
         /// Returns the source file and position where the item at the given position is declared at,
         /// if such a declaration exists, and returns null otherwise.
         /// </summary>
-        public static Location DefinitionLocation(this FileContentManager file, CompilationUnit compilation, Position position)
+        public static Location? DefinitionLocation(this FileContentManager file, CompilationUnit compilation, Position? position)
         {
             var symbolInfo = file?.TryGetQsSymbolInfo(position, true, out CodeFragment _); // includes the end position
-            if (symbolInfo == null || compilation == null)
+            if (file is null || symbolInfo == null || compilation == null || position == null)
```

Comment:
If the types for symbolInfo, compilation, and position do not overload the `==` operator, could we use `is null` for them for consistency?